recipes/solexaqa/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "solexaqa" %}
+{% set version = "3.1.7.1" %}
+{% set sha256 = "b3bdabf955387a4f6176dbdfda1cf03bb869c077b4eec152304f2e76be5f0cf6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [osx or win]
+
+source:
+  url: https://downloads.sourceforge.net/project/solexaqa/src/SolexaQA++_v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - gxx_linux-64 >=7.3.0
+    - make
+  host:
+    - boost
+    - icu
+    - libstdcxx-ng
+    - zlib
+  run:
+    - boost
+    - icu
+    - libstdcxx-ng
+    - zlib
+    # R is not a strict dependency, but can provide enhanced functionality,
+    # so we leave it as an option for the user, not a hard requirement.
+    # - r
```

Comment:
r-base would be one alternative. I think we agreed to include optional dependencies by default - also to make the containers work by default.

Reply:
I think in 99% of the uses of SolexaQA, R is not used - maybe even 5 9s ;-). Even r-base is not a small package.  My strong preference is to omit it.